src/test/java/cloud/eppo/BaseEppoClientBanditTest.java

```diff
@@ -62,82 +63,21 @@ public void reset() {
     clearInvocations(mockAssignmentLogger);
     clearInvocations(mockBanditLogger);
     doNothing().when(mockBanditLogger).logBanditAssignment(any());
-    BaseEppoClient.getInstance().setIsGracefulFailureMode(false);
+    eppoClient.setIsGracefulFailureMode(false);
   }
 
   @ParameterizedTest
   @MethodSource("getBanditTestData")
   public void testUnobfuscatedBanditAssignments(File testFile) {
-    BanditTestCase testCase = parseTestCaseFile(testFile);
-    runBanditTestCase(testCase);
-  }
-
-  private static Stream<Arguments> getBanditTestData() {
```

Comment:
This method and the next two have been moved to` BanditTestCase`. This cleans up this file and makes it so they can be used upstream.